stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -0,0 +1,25 @@
+package io.getstream.chat.ui.sample.application
+
+import android.app.Application
+import io.getstream.chat.ui.sample.data.user.UserRepository
+
+class App : Application() {
+
+    lateinit var chatInitializer: ChatInitializer
+    lateinit var userRepository: UserRepository
+
+    override fun onCreate() {
+        super.onCreate()
+        instance = this
+
+        chatInitializer = ChatInitializer(this)
+        userRepository = UserRepository()
+
+        DebugMetricsHelper().init()
+    }
+
+    companion object {
+        lateinit var instance: App
+            private set
```

Comment:
Might be worth a comment to recommend using a DI library (not a specific one, just any) in real applications instead of this approach 🙂